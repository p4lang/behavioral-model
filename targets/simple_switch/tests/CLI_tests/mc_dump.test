#!/bin/sh
./run_one_test.py $srcdir/testdata mc_dump ecmp.json --pre SimplePreLAG
