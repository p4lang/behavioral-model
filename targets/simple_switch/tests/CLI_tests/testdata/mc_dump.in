mc_mgrp_create 1
mc_node_create 2 1 5 8 | 1 3
mc_node_associate 1 0
mc_node_create 4 5 8 | 1 2
mc_node_associate 1 1
mc_set_lag_membership 1 6 9 12
mc_dump
