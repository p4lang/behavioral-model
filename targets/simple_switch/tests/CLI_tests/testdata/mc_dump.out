????
Creating multicast group 1
????
Creating node with rid 2 , port map 100100010 and lag map 1010
node was created with handle 0
????
Associating node 0 to multicast group 1
????
Creating node with rid 4 , port map 100100000 and lag map 110
node was created with handle 1
????
Associating node 1 to multicast group 1
????
Setting lag membership: 1 <- 1001001000000
????
==========
MC ENTRIES
**********
mgrp(1)
  -> (L1h=0, rid=2) -> (ports=[1, 5, 8], lags=[1, 3])
  -> (L1h=1, rid=4) -> (ports=[5, 8], lags=[1, 2])
==========
LAGS
lag(1) -> ports=[5, 8]
==========
????
